2.4/README.md

```diff
@@ -58,6 +58,14 @@ will be started. If you are re-attaching the volume to another container, the
 creation of the database user and admin user will be skipped and only the
 MongoDB daemon will be started.
 
+Custom configuration file
+---------------------------------
+
+It is allowed to use custom configuration file for mongod server.
+
+To use custom configuration file in container it has to be mounted into `/etc/mongod.conf`. For example to use configuration file stored in `/home/user` directory use this option for `docker run` command: `-v /home/user/mongod.conf:/etc/mongod.conf:Z`.
```

Comment:
"Providing a custom configuration file supercedes the individual configuration environment variable values."